sdk/identity/identity/CHANGELOG.md

```diff
@@ -17,6 +18,7 @@
   - Added `regionalAuthority` property to `ClientSecretCredentialOptions` and `ClientCertificateCredentialOptions`.
   - If instead of a region, `AutoDiscoverRegion` is specified as the value for `regionalAuthority`, MSAL will be used to attempt to discover the region.
   - A region can also be specified through the `AZURE_REGIONAL_AUTHORITY_NAME` environment variable.
+- Now, the `CP1` client capabilities will be enabled by default for all of the credentials using MSAL. This can be disabled by setting the environment variable `AZURE_IDENTITY_DISABLE_CP1` (any value counts).
```

Comment:
What does this actually? Should a user expect new/different observable behavior by default?

Reply:
This is for CAE challenges, you can see Schaab’s draft here: https://gist.github.com/schaabs/cdf28cc3811072e47bce9af417e3c03f

Reply:
@sadasant 

I still don't understand. These are not common acronyms to my knowledge, so I think some more explanation would be helpful, especially for customers. Do we expect customers to implicitly understand continuous access evaluation, or is that understanding even necessary?

Basically, from a customer perspective, what are CP1 capabilities and what is a CAE challenge? Is there something that a customer should expect to be different after this commit? Under which circumstances should a user set `AZURE_IDENTITY_DISABLE_CP1`?

Reply:
Client capabilities is only relevant for the Azure resources that support it. If they support it, we want to make sure they send the challenge back so that our challenge callbacks can process them. This should be fine in general (which is why we’re enabling this by default), and if it becomes a problem, Azure clients can disable this behavior.

Here are more docs regarding this part of the Azure specs: https://docs.microsoft.com/en-us/azure/active-directory/develop/claims-challenge#client-capabilities

For the Identity SDK, this PR is more of an alignment with other languages, since they have already released with these changes.

Reply:
I spoke with Will and I understood how to approach this! I’ll update this PR.

Reply:
How does this look? https://github.com/Azure/azure-sdk-for-js/pull/15390/commits/ce4742d2ec42c8f94dd88760ce76c7f10862d093